src/Video.tsx

```diff
@@ -459,6 +454,20 @@ const Video = forwardRef<VideoRef, ReactVideoProps>(
       [onAspectRatio],
     );
 
+    const _onPlaybackRateChange = useCallback(
```

Comment:
can you please move back _onPlaybackRateChange at its initial position in the file ? no need for you to be owner of this function :)